### DISCUSSION TITLE
Gizmo board port map

### DISCUSSION THREAD

**Discussion by jds-designs** [gizmo-platform/gizmo#40] [General]
Here is a guide I designed showing the ports on the Gizmo board labeled with their names from the current version of the Gizmo library.

![Gizmo Ports v4](https://github.com/user-attachments/assets/a7b9ac8b-4d5e-47a9-9159-027cf565e12b)

Note that the right column of UART ports should not be used, as it is currently not mapped in the Gizmo library. The Gizmo development team will most likely change this in the future.

**Comment by Vaelatern** [comment]
Could you point at the pins actually referenced perhaps, instead of the ground pin?

I know this makes it harder because the pins referenced are on the inside of the board, and ground is around the outside.

**Reply by jds-designs** [reply]
Good point. I just replaced the image with an updated version that has the lines going over to what I believe are the correct pin holes. Hopefully it's still clear.
